project-base/src/Shopsys/ShopBundle/Controller/Front/ErrorController.php

```diff
@@ -127,6 +136,7 @@ private function createErrorPageResponse($statusCode)
             $errorPageStatusCode
         );
 
+        $errorPageContent = str_replace('{{ERROR_ID}}', $this->errorIdProvider->getErrorId(), $errorPageContent);
```

Comment:
What about moving this into `ErrorPagesFacade::getErrorPageContentByDomainIdAndStatusCode`? 
Upgrade notes will be much simpler, but it's fair to say, the actual replace is done only if the template is changed during upgrade.

Suggestion to consider, not forcing it :)